src/pages/get-started/api-security.md

```diff
@@ -7,6 +7,217 @@ This topic describes best practices for [API security](https://owasp.org/www-pro
 
 # API security
 
+## Rate limiting for payment information endpoint and mutation
+
+In a carding attack, an attacker tries to validate which credit cards are valid, usually in batches of thousands. In some cases, this can also be used to brute force missing details like expiry date. Adobe Commerce merchants suffer from this attack type, as their shops and integrations with 3rd party payment gateways are abused.
+Adobe Commerce introduces a new configuration for adding rate limiting for payment information endpoint and mutation. This is an added layer of protection for merchants to prevent/slow carding attacks that test many stolen credit cards at once.
+Rate limiting is disabled by default but can be enabled and configured via CLI and/or using UI. The threshold can be configured by interval and can be set independently for customers and guests.
+
+### How the feature works?
+This rate limiting functionality will be affected to the next entry points:
+- WebAPI:
+    - `{{base_url}}/rest/V1/guest-carts/{{cart_id}}/payment-information`
+    - `{{base_url}}/rest/V1/guest-carts/{{cart_id}}/order`
+    - `{{base_url}}/rest/V1/carts/mine/payment-information`
+    - `{{base_url}}/rest/V1/carts/mine/order`
+- GraphQL: `{{base_url}}/graphql`
+```
+mutation {
+    placeOrder(input:{
+        cart_id:"{{cart_id}}"
+    }) {
+    order {order_number}
+    }
+}
+```
+- Module InstantPurchase `magento/module-instant-purchase`
+
+The configuration and default values are located in the path `app/code/Magento/Quote/etc/config.xml`
+```xml
+<?xml version="1.0"?>
+<!-- ... -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        ...
+        <sales>
+            <backpressure>
+                <enabled>0</enabled>          <!-- Enable rate limiting for placing orders -->
+                <limit>10</limit>             <!-- Requests limit per authenticated customer -->
+                <guest_limit>50</guest_limit> <!-- Requests limit per guest -->
+                <period>60</period>           <!-- Counter resets in a ... [sec.]-->
+            </backpressure>
+        </sales>
+    </default>
+</config>
+```
+These settings can be explained as follows:
+- Sales restrictions are enabled `sales/backpressure/enabled` = `1`.
```

Comment:
Is this binary?

Reply:
No, I just to try explain settings

Reply:
@jhadobe 
Is this not relevant anymore?